src/main/play-doc/api/BundleAPI.md

```diff
@@ -81,7 +81,7 @@ Content-Type: application/json
 ]
 ```
 
-Where `10.11.23.22:7001` and `10.11.23.22:7002` are the host and port of the service which is currently running.
+Where `10.11.23.22:7001` and `10.11.23.22:7002` are the host and host port of the service which is currently running.
```

Comment:
@huntc - `host port` is the bound port of the service which is currently running, isn't it?